custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -420,8 +420,11 @@ def create_partition(self, state_id):
 
     def aggregation_queries(self):
         return [
-            """INSERT INTO "{new_tablename}" (SELECT * FROM "{tmp_tablename}")""".format(new_tablename=self.new_tablename, tmp_tablename=self.temporary_tablename),
+            """INSERT INTO "{new_tablename}" (SELECT * FROM "{tmp_tablename}")""".format(
+                new_tablename=self.new_tablename, tmp_tablename=self.temporary_tablename),
             'DROP TABLE IF EXISTS "{monthly_tablename}"'.format(monthly_tablename=self.monthly_tablename),
-            """ALTER TABLE "{new_tablename}" RENAME TO \"{tablename}\"""".format(new_tablename=self.new_tablename, tablename=self.monthly_tablename),
-            """ALTER TABLE "{tablename}" ATTACH PARTITION "{monthly_tablename}" FOR VALUES IN ('{month}')""".format(monthly_tablename=self.monthly_tablename, month=self.month.strftime('%Y-%m-%d'), tablename=self.tablename),
+            """ALTER TABLE "{new_tablename}" RENAME TO \"{tablename}\"""".format(
```

Comment:
this change looks not required. can you ensure to revert these ones
@ShubhamBansal1997